integration-tests/pom.xml

```diff
@@ -13,6 +13,7 @@
   <artifactId>integration-tests</artifactId>
   <version>0.0.1-SNAPSHOT</version>
   <name>Event Bridge :: Integration Tests</name>
+  
```

Comment:
Please remove.

Reply:
This was to check that the parallel testing does work on `Manager - CI / Build all images and run E2E tests (pull_request_target)`. Else the check does not trigger. Gonna remove that.